static/scripts/onboarding/onboarding.ts

```diff
@@ -22,107 +24,23 @@ const chainIdSelect = document.getElementById("chainId") as HTMLSelectElement;
 const loader = document.querySelector(".loader-wrap") as HTMLElement;
 
 const APP_ID = 236521;
-const DEFAULT_ORG = "ubiquity";
 const REPO_NAME = "ubiquibot-config";
-const DEFAULT_REPO = "ubiquibot";
 const KEY_PATH = ".github/ubiquibot-config.yml";
-const DEFAULT_PATH = "ubiquibot-config-default.json";
-const KEY_NAME = "private-key-encrypted";
+const PRIVATE_ENCRYPTED_KEY_NAME = "privateKeyEncrypted";
+const EVM_NETWORK_KEY_NAME = "evmNetworkId";
 const KEY_PREFIX = "HSK_";
 const X25519_KEY = "5ghIlfGjz_ChcYlBDOG7dzmgAgBPuTahpvTMBipSH00";
```

Comment:
I guess this should be a variable because this is associated with each funding wallet (organizations are able to have funding custom wallets down to the repository level)
```suggestion
const X25519_PUBLIC_KEY = "5ghIlfGjz_ChcYlBDOG7dzmgAgBPuTahpvTMBipSH00";
```

@rndquu rfc

Reply:
I don't fully understand what you mean. Renaming from `X25519_KEY` to `X25519_PUBLIC_KEY ` is more explicit, I agree.

> this is associated with each funding wallet

`X25519_PUBLIC_KEY` is the same for all partner wallets. They use this `X25519_PUBLIC_KEY` to encrypt their evm private keys which we are able to decrypt on the bot's side using `X25519_PRIVATE_KEY`.

Reply:
You're correct, there's only a single X25519 key in our system